views/components/packages/addPackage.jsx

```diff
@@ -29,6 +30,21 @@ const AddPackage = ({ dispatch, id, isAdding }) => {
           }}
         />
       </div>
+      <select
+        className="form-control left"
+        ref={node => {
+          if (node) {
```

Comment:
This was a strange bug I found that happened consistently. It fetches the ref twice - once before onChange and once after onChange. The second time it will be null for some reason and override the value that was set in onChange.